src/thunderbots/shared_util/constants.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+#include "../software/ai/world/team.h"
+
+// Constants for ROS nodes, message, and topics
+static const std::string BACKEND_INPUT_BALL_TOPIC          = "backend/ball";
+static const std::string BACKEND_INPUT_FIELD_TOPIC         = "backend/field";
+static const std::string BACKEND_INPUT_FRIENDLY_TEAM_TOPIC = "backend/friendly_team";
+static const std::string BACKEND_INPUT_ENEMY_TEAM_TOPIC    = "backend/enemy_team";
+static const std::string AI_PRIMITIVES_TOPIC               = "backend/enemy_team";
+
+// TODO: Make this a tuneable parameter
+const TEAM_COLOUR FRIENDLY_TEAM_COLOUR = YELLOW;
```

Comment:
Ok, I look at `TEAM_COLOUR` and immediately think that that the type should not be all caps.
But then there's stuff like `STPHL` and `HL` that would look awkward if not all-caps, and so on...
Maybe this comment should be wrapped with a huge NIT warning.

Reply:
Yea this is tricky, because I can see the team colour as both an enum, which should be CamelCase, but in some ways it's also a constant (or set of constants) and therefore should be uppercase.

Maybe we could add another rule to variable naming that says something like "all acronyms are capitalized", and that would then cover HL and sTPHL.

I'm fine changing TEAM_COLOUR to CamelCase. Thoughts on the acronyms?

Reply:
Yeah, I think TEAM_COLOUR should be camel case, since it's not an initialism like STPHL and HL and should be an enum type.